code/modules/mob/living/silicon/laws.dm

```diff
@@ -5,6 +5,11 @@
 	if (!laws)
 		make_laws()
 
+/mob/living/silicon/proc/log_current_laws()
+	var/list/the_laws = laws.get_law_list(include_zeroth = TRUE)
+	var/lawtext = the_laws.Join("<br/>")
```

Comment:
I believe the log files you're using aren't rendered as HTML, they're rendered as plain text. As a result, joining the laws with HTML line breaks, well... Does what it does in your example screenshot. Having them all on one line isn't bad because it's convenient for automated log file parsers and grep users. What's your thoughts on this?

Reply:
I changed it a little, looks like this now
It's all still on one line, which I somewhat don't like, but I also don't know if it should be on separate ones either since it might get mixed with other logged things.
![image](https://user-images.githubusercontent.com/53777086/140842227-e68a6309-3847-4b46-a61a-5e6910774b5f.png)